lib/ZcapClient.js

```diff
@@ -138,14 +144,24 @@ export class ZcapClient {
 
     const signedDelegatedCapability = await jsigs.sign(
       delegatedCapability, {
-        suite: new Ed25519Signature2018({
+        suite: new Ed25519Signature2020({
           signer: this.delegationSigner,
           verificationMethod: this.delegationSigner.id
         }),
         purpose: new CapabilityDelegation({
           capability,
           capabilityChain: (url) ? [await generateZcapUri({url})] : undefined
-        })
+        }),
+        // FIXME: WHAT'S THE SOLUTION FOR DOCUMENTLOADER?
```

Comment:
Pass a doc loader that will properly return the ed25519-2020 context.

Reply:
@dlongley @dmitrizagidulin I would really like to see the ergonomics of all this improved.  Can we imagine a situation where the SignatureSuites can provide a document loader for their context and this gets automagically added to a documentLoader that is internal to jsonld-signatures.  

I also feel that having to deal with putting a context on my document specific to the signature suite is work that could be done elsewhere.  On thought that occurred to me was... Is it not possible for the `proof` to have its own context?  And therefore consumers of the `sign` API don't have to correlate the context on their documents with the signature suites.